README.md

```diff
@@ -167,9 +167,19 @@ So, for example, to get my [git deck](https://github.com/Stvad/Software_Engineer
 ## Snapshots
 
 **CrowdAnki** can help you preserve **the history of edits for your decks**.  
-It does this by exporting them in a specified location and creating a git commit each time you do a snapshot.
-
-You can take snapshots manually via `File > CrowdAnki: Snapshot` menu action.  
-Or you can enable automated snapshots in plugin configuration. You can find more details on how to do it [here](crowd_anki/config.md) 
-
-Other things you can [configure](crowd_anki/config.md) is the location of the snapshot and what decks should be included into it. 
+It does this by exporting them in a specified location and creating a git commit each time you do a snapshot. 
```

Comment:
why is this updateted? are you introducing windows line endings? 🧐

Reply:
This was just the deletion of lines 171-175. Line 170 is unchanged :thinking: silly Github (also I do all my dev on Linux fyi :wink:)